### ISSUE TITLE
Word wrap separates between characters that fit together

### ISSUE THREAD

**Issue by askebv** [slynch8/10x#2204]
As shown in the image, it currently separates between `&` and the variable name in C++:
![image](https://github.com/slynch8/10x/assets/4981659/eb4c87f9-fcb7-4448-9649-4f75d9ba7774)

Ideally I think there should be different separators per language (possibly even user configurable). But the easiest generalization might be to separate on `,`, `(`, `{` and ` `(whitespace).

**Comment by vulcan-dev** [comment]
Yeah, I agree. If it's something like `*x` or `&x`, even casts, I think that should all count as one word.
